### PR TITLE
add new maps

### PR DESCRIPTION
**Pull request by hyuunnn**

### CODE REVIEW

File: evtx/Maps/Application_10002.map [hunk 1]
```diff
@@ -0,0 +1,65 @@
+Author: Hyun Yi @hyuunnn
+Description: Terminated due to non-response
+EventId: 10002
+Channel: "Application"
+Maps:
```
Comment (rathbuna) [inline evtx/Maps/Application_10002.map:5]: Can you add a Provider for the Application log maps? There are many Providers with overlapping event IDs sometimes so it's best practice to ensure Provider is populated for Application.evtx maps.

File: evtx/Maps/System_6008.map [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+EventId: 6008
+Channel: "System"
+Maps:
+  -
+    Property: PayloadData1
+    PropertyValue: "Time, Date : %Data%"
+    Values:
+      -
+        Name: Data
+        Value: "/Event/EventData/Data"
```
Comment (rathbuna) [inline evtx/Maps/System_6008.map:12]: Did you test this? Did it grab data? It looks different than most other maps on this line.

**Review by rathbuna — CHANGES_REQUESTED** [review]
FYI Security 4723 already exists as of yesterday. Please review that one and ensure everything is covered. Make changes as you see fit. Two eyes are better than one on any map(s). Thanks for your contributions, keep it up!

**Comment by hyuunnn** [comment]
Oh sorry.. I'll clean up and upload again.

File: evtx/Maps/System_6008.map (hunk 2 above)
Comment (hyuunnn) [reply evtx/Maps/System_6008.map:12]: image

It works correctly.

File: evtx/Maps/Application_10002.map (hunk 1 above)
Comment (hyuunnn) [reply evtx/Maps/Application_10002.map:6]: Fixed :)

**Review by rathbuna — APPROVED** [review]

**Pull request merged**